.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+
+node_js:
+  - "6.2"
+
+before_script:
+  - npm install -g gulp bower
+  - bower install
```

Comment:
The default is `npm test`, which means it runs the script listed in `package.json` under `scripts.test`.  Which in the case of this repo is `gulp qa`.  So specifying the `script` in `.travis.yml` is unnecessary, since the default does what we want.